libs/test-common/src/main/java/org/codice/ddf/test/common/AbstractComponentTest.java

```diff
@@ -73,7 +74,8 @@
     return options(
         getContainerOptions().get(),
         getApplicationOptions(portFinder).get(),
-        getTestBundleOptions().build());
+        getTestBundleOptions().build(),
+        defaultMavenRepositoriesOption());
```

Comment:
Does anyone know if there is a reason we wouldn't want this to happen for all component tests. It definitely fixes problems with incremental builds. I still hope to find a way to make it use only the local maven repo since maven should have already pulled down all the required dependencies by the time the container is launched I think